db/db_range_del_test.cc

```diff
@@ -49,6 +49,10 @@ TEST_F(DBRangeDelTest, FlushOutputHasOnlyRangeTombstones) {
   ASSERT_EQ(1, NumTableFilesAtLevel(0));
 }
 
+TEST_F(DBRangeDelTest, BasicCallDeleteRangeWithoutCF) {
+  ASSERT_OK(db_->DeleteRange(WriteOptions(), "dr1", "dr2"));
+}
```

Comment:
I don't think this tests anything other than making sure that the new api can be called. So, may be, it can totally be removed?